src/videojs-fairplay.js

```diff
@@ -212,12 +237,38 @@ class Html5Fairplay {
       status,
     } = event.target;
 
+    const {
+      licenseResponseHeaders,
+      licenseResponseErrorContent,
+    } = this.protection_;
+
     if (status !== 200) {
       this.onRequestError(event.target, ERROR_TYPE.FETCH_LICENCE);
 
+      // Return response content
+      if (licenseResponseErrorContent && typeof licenseResponseErrorContent === 'function') {
```

Comment:
`licenseResponseErrorContent` => `licenseLoadError` is there a better name or something more applicable?

is an inverse needed? `licenseLoadSuccess`? `licenseLoadHeaders`?